<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgSettings</class>
 <widget class="QDialog" name="DlgSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="resetButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	color:rgbrgb(0, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>恢复默认</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="colorGroupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>249</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>颜色</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QRadioButton" name="radioButtonRed">
      <property name="styleSheet">
       <string notr="true">QRadioButton
{
	background-color: rgb(255, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>红</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonGreen">
      <property name="styleSheet">
       <string notr="true">QRadioButton
{
	background-color: rgb(0, 255, 0);
}</string>
      </property>
      <property name="text">
       <string>绿</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonBlue">
      <property name="styleSheet">
       <string notr="true">QRadioButton
{
	background-color: rgb(0, 85, 255);
}</string>
      </property>
      <property name="text">
       <string>蓝</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonYellow">
      <property name="styleSheet">
       <string notr="true">QRadioButton
{
	background-color: rgb(255, 255, 0);
}</string>
      </property>
      <property name="text">
       <string>黄</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButtonBlack">
      <property name="styleSheet">
       <string notr="true">QRadioButton
{
	color:rgb(255, 255, 255);
	background-color: rgb(0, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>黑</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="text">
       <string>粗细</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="thicknessComboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <item>
       <property name="text">
        <string>细</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>中</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>粗</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>214</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="confirmButton">
      <property name="text">
       <string>确认</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>121</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>字号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="fontSizeComboBox">
      <item>
       <property name="text">
        <string>小</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>中</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>大</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
